cmd/tikv-ctl/src/main.rs

```diff
@@ -1050,867 +1051,661 @@ fn warning_prompt(message: &str) -> bool {
-                        .help("input file path"),
-                )
-                .arg(
-                    Arg::with_name("out-file")
-                        .long("out-file")
-                        .takes_value(true)
-                        .required(true)
-                        .help("output file path"),
-                ),
-        )
-        .subcommand(
-            SubCommand::with_name("encryption-meta")
-                .about("Dump encryption metadata")
-                .subcommand(
-                    SubCommand::with_name("dump-key")
-                        .about("Dump data keys")
-                        .arg(
-                            Arg::with_name("ids")
-                                .long("ids")
-                                .takes_value(true)
-                                .use_delimiter(true)
-                                .help("List of data key ids. Dump all keys if not provided."),
-                        ),
-                )
-                .subcommand(
-                    SubCommand::with_name("dump-file")
-                        .about("Dump file encryption info")
-                        .arg(
-                            Arg::with_name("path")
-                                .long("path")
-                                .takes_value(true)
-                                .help("Path to the file. Dump for all files if not provided."),
-                        ),
-                ),
-            )
-        .subcommand(
-            SubCommand::with_name("bad-ssts")
-                .about("Print bad ssts related infos")
-                .arg(
-                    Arg::with_name("db")
-                        .long("db")
-                        .required(true)
-                        .takes_value(true)
-                        .help("db directory."),
-                )
-                .arg(
-                    Arg::with_name("manifest")
-                        .long("manifest")
-                        .takes_value(true)
-                        .help("specify manifest, if not set, it will look up manifest file in db path"),
-                )
-                .arg(
-                    Arg::with_name("pd")
-                        .required(true)
-                        .long("pd")
-                        .takes_value(true)
-                        .value_delimiter(",")
-                        .help("PD endpoints"),
-                )
-        );
+    tikv::tikv_version_info(build_timestamp)
+});
+
+const RAW_KEY_HINT: &str = "Raw key (generally starts with \"z\") in escaped form";
+
+#[derive(StructOpt)]
+#[structopt(
+    name = "TiKV Control (tikv-ctl)",
+    about = "A tool for interacting with TiKV deployments.",
+    author = crate_authors!(),
+    version = &**VERSION_INFO,
+    long_version = &**VERSION_INFO,
+    setting = AppSettings::DontCollapseArgsInUsage,
+)]
+struct Opt {
+    #[structopt(long)]
+    /// Set the address of pd
```

Comment:
Maybe we should use `#[structopt(short, help = "xxx")]`, otherwise the help message won't show in command line.

Reply:
Doc strings and `help = xxx` are equivalent. See [related chapter](https://docs.rs/structopt/0.3.22/structopt/#help-messages) of its document.